static/js/reducers/settings.js

```diff
@@ -2,10 +2,10 @@
 import * as api from "../lib/api"
 import { GET, PATCH } from "redux-hammock/constants"
 
-export const FREQUENCY_IMMEDIATE = "immediate"
-export const FREQUENCY_DAILY = "daily"
-export const FREQUENCY_WEEKLY = "weekly"
-export const FREQUENCY_NEVER = "never"
+export const FREQUENCY_IMMEDIATE: "immediate" = "immediate"
```

Comment:
This is to tell flow that it can use these as constants across module boundaries? Should we do this for any of the other constants?

Reply:
yeah, basically what it does is make `FREQUENCY_DAILY` actually have type `"daily"` instead of type `string`. then we can use the types of the constants in a union type to have stricter checking on string constant options.

I think we should use this anywhere that we have a strictly limited set of constant strings that a variable or property could hold (for instance some of the properties in API responses might qualify). One place I used it recently was the `imageSize` prop for the profile image, which is now type-checked by flow to be `"micro" | "small" | "medium"`. I'll take a quick look for other places we could take advantage of this rn.

Reply:
I just made a few more changes along those lines for some of the LinkType business, there may be a few more places where we can take advantage of this stuff more, but I think we can just try to make those changes going forward as we notice them